lib/saito/ui/game-playerbox/game-playerbox.js

```diff
@@ -7,8 +7,44 @@
           this defaults to status for the player and is used by some games for interactive controls
     --- a Div for graphical elements, such a dealt hand of cards
 
-    Use render to create the playerboxes
-    Use attachEvents to make them movable
+  API:
```

Comment:
I might confuse this, but I assume if you stick to the [JSDoc](https://jsdoc.app/) format this woudl be auto-generated to here: https://saitotech.github.io/saito-lite/index.html  

This would make it more visible if other devs look at the created docs.